src/core/Button/Button.tsx

```diff
@@ -1,34 +1,44 @@
-import React, { Component } from 'react';
+import React, { Component, ReactNode } from 'react';
 import { default as styled } from 'styled-components';
 import classnames from 'classnames';
 import { TokensProp, InternalTokensProp } from '../theme';
 import { withSuomifiDefaultProps } from '../theme/utils';
 import { baseStyles } from './Button.baseStyles';
-import {
-  Button as CompButton,
-  ButtonProps as CompButtonProps,
-} from '../../components/Button/Button';
+import { HtmlButton, HtmlButtonProps } from '../../reset';
 import { Icon, IconProps, BaseIconKeys } from '../Icon/Icon';
-import { UnstyledButton } from './UnstyledButton';
 
 type ButtonVariant =
   | 'default'
   | 'inverted'
   | 'secondary'
   | 'secondary-noborder'
-  | 'tertiary'
-  | 'unstyled';
-
```

Comment:
Just for readability, might be good to leave the empty row between `type` and `export interface` here.

Reply:
Fixed.